snorkel/labeling/preprocess/nlp.py

```diff
@@ -39,7 +39,8 @@ def __init__(
         language: str = EN_CORE_WEB_SM,
         disable: List[str] = DEFAULT_DISABLE,
     ) -> None:
-        super().__init__(dict(text=text_field), dict(doc=doc_field))
+        name = type(self).__name__
```

Comment:
Why not just set this as the default for `name`?

Reply:
You mean allow people to pass in their own name with this as the default if None is passed in?